drizzlepac/haputils/processing_utils.py

```diff
@@ -341,6 +341,55 @@ def compute_sregion(image, extname='SCI'):
     if closefits:
         hdu.close()
 
+def add_skycell_to_header(image_filename, extname='SCI'):
+    """Determines the skycells for which the image falls within and adds the 
+    information to the science header.
+
```

Comment:
Please add a note here that indicates the SKYCELL value for IPPPSSOOT and SVM products may be different as the current computation is based upon the WCSNAME of the input images.

This is redundant with the comment in runastrodriz.py but we need to make sure it is well-documented.